src/ProgressOnderwijsUtils/RandomHelper.cs

```diff
@@ -14,6 +15,13 @@ public sealed class RandomHelper
         public static RandomHelper Insecure(int seed)
             => new RandomHelper(new Random(seed).NextBytes);
 
+        [NotNull]
+        public static RandomHelper ImplicitlyInsecure([CallerLineNumber] int linenumber = -1, [CallerFilePath] string filepath = "", [CallerMemberName] string membername = "")
+            => Insecure(GetNaiveHashCode(filepath) ^ GetNaiveHashCode(membername) ^ linenumber);

```

Comment:
Ik denk dat je beter linenumber weg kan laten ivm autoformat verassingen.  Namen passen typisch minder indirect aan.

Maar dat is meer een gokje dan heel belangrijk, wmb; iig is het zo reproduceerbaar, en dat is verreweg het meest belangrijke.

Reply:
Oh, en omdat je build-path onafhankelijk wilt zijn is iets as `Path.GetFileName(filepath)` een goed idee.

Netto dus iets als...
```suggestion
            => Insecure(GetNaiveHashCode(System.IO.Path.GetFileName(filepath)) + 1337 * GetNaiveHashCode(membername));
```

oid?  (ik heb ook `^` vervangen met een non-symmetrische operator, omdat dat typisch een beter idee is in hashcodes)

Reply:
👍 👍